test_docker.sh

```diff
@@ -1 +0,0 @@
-docker run --rm -v $(pwd):/Qt.py mottosso/qt.py27
```

Comment:
Yes, sure. Do you want to only keep Python 2.7 in there?
It would have to be:

```
docker run --rm -v $(pwd):/Qt.py -e PYTHON=2.7 fredrikaverpil/qt.py:2017
```

These are also available:

```
docker run --rm -v $(pwd):/Qt.py -e PYTHON=3.4 fredrikaverpil/qt.py:2017
docker run --rm -v $(pwd):/Qt.py -e PYTHON=3.5 fredrikaverpil/qt.py:2017
```